<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioEffects</class>
 <widget class="QMainWindow" name="AudioEffects">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AudioEffects</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>music/Music.png</normaloff>music/Music.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MPL_Widget" name="plotWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="Phonon::SeekSlider" name="seekSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>581</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="iconVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>291</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="playToolButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>music/Play.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stopToolButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;停止&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>music/Stop.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Phonon::VolumeSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="MPL_Widget" name="plotWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="fileOpenAction"/>
    <addaction name="fileExitAction"/>
   </widget>
   <widget class="QMenu" name="menuAudioEffects">
    <property name="title">
     <string>&amp;Effects</string>
    </property>
    <addaction name="actionWavesReverb"/>
    <addaction name="actionLowFilter"/>
    <addaction name="actionEcho_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAudioEffects"/>
  </widget>
  <action name="fileOpenAction">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="fileExitAction">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionWavesReverb">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WavesReverb</string>
   </property>
  </action>
  <action name="actionLowFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LowFilter</string>
   </property>
  </action>
  <action name="actionEcho_2">
   <property name="text">
    <string>Echo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
  <customwidget>
   <class>MPL_Widget</class>
   <extends>QWidget</extends>
   <header>mpl_pyqt4_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
